lighthouse-core/test/gather/computed/pushed-requests-test.js

```diff
@@ -7,15 +7,17 @@
 
 /* eslint-env mocha */
 
-const GathererClass = require('../../../gather/computed/pushed-requests');
+const Runner = require('../../../runner.js');
```

Comment:
ok, so I'll wait until Paul gets back to delete, but also don't care enough to update :)

I think it's fine with `instatiateComputedArtifacts`, and it's annoying enough to recreate that network record in a devtoolsLog (that site doesn't appear to be using http2 push anymore?) that mocking `requestNetworkRecords` doesn't seem like the worrrrst thing.

wdyt?